apicast/bin/apicast

```diff
@@ -110,6 +110,7 @@ done
 
 main+=("daemon ${daemon};")
 main+=("worker_processes ${worker_processes};")
+main+=$(printenv | awk '$1 ~ /^(APICAST|THREESCALE)_/ {split($0,env,"="); print "env", env[1] ";"}')
```

Comment:
1. `bin/apicast` is the only way to start apicast. It was before anyway. It sets up daemon mode, worker processes and everything. It finds the correct path etc etc etc. In the future we might have to template the whole nginx config, so you can forget there is nginx now.

2. It is error prone to add everything into the configuration. Tests can be passing but without full integration suite there is no way to actually verify it. There never was single source of truth because code evaluated in the init phase had access to all the variables (even not defined) where code executed later would get them stripped. That was super confusing and not predictable. 

3. I don't see a way to get dynamic variables working for https://github.com/3scale/apicast/pull/293. Unless we figure out some super complicated format like: `id:version;id2:version2`.